refinedstorage2-platform-fabric/src/main/java/com/refinedmods/refinedstorage2/platform/fabric/integration/recipemod/rei/RefinedStorageModPlugin.java

```diff
@@ -0,0 +1,24 @@
+package com.refinedmods.refinedstorage2.platform.fabric.integration.recipemod.rei;
+
+import com.refinedmods.refinedstorage2.platform.api.PlatformApi;
+import com.refinedmods.refinedstorage2.platform.api.integration.recipemod.IngredientConverter;
+
+import me.shedaniel.rei.api.client.plugins.REIClientPlugin;
+import me.shedaniel.rei.api.client.registry.screen.ScreenRegistry;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
+public class RefinedStorageModPlugin implements REIClientPlugin {
```

Comment:
```suggestion
public class RefinedStorageREIClientPlugin implements REIClientPlugin {
```
Important to respect the `{ImplName}{InterfaceName}`  format.